actnow/profiles/validators.py

```diff
@@ -0,0 +1,11 @@
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+
+def validate_social_media_link(link):
+    link = link.lower()
+    if not any(["twitter" in link, "facebook" in link, "whatsapp" in link]):
```

Comment:
Check for full domain i.e. `facebook.com/` or `twitter.com/` (not sure about whatsapp). Otherwise something like `myfaketwitterbutnot.com` will pass the validation.